repos/system_upgrade/el7toel8/actors/yumupgradability/actor.py

```diff
@@ -0,0 +1,39 @@
+from subprocess import CalledProcessError
+
+from leapp.actors import Actor
+from leapp.exceptions import CommandError
+from leapp.tags import IPUWorkflowTag, ExperimentalTag, PreparationPhaseTag
+from leapp.libraries.stdlib import call
+
+
+class YumUpgradability(Actor):
+    """
+    Copy yum v3 configuration to yum v4 /etc/dnf directory
+
+    Steps
+        1) Create /etc/dnf directory
+        2) Copy yum data to the /etc/dnf directory
+        3) Make symlinks under /etc/yum
+    """
+
+    name = 'yum_upgradability'
+    consumes = ()
+    produces = ()
+    tags = (IPUWorkflowTag, PreparationPhaseTag, ExperimentalTag)
+
+    script_path = '../../tools/copy_yum_v3_configuration.sh'
```

Comment:
@vinzenz There is API function that returns correct path, isn't?

Reply:
Removed the relative path. As this should be in the executable path.